brainvisa/toolboxes/data management/processes/import/FLAIRmri/ImportFLAIRMRI.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+#  This software and supporting documentation are distributed by
```

Comment:
This header seems obsolete to me. I am biased because I do not like such headers in code but I would simply remove it completely.

Reply:
Okay for me. 
This process is the one coded by Ludovic a long time ago, I do not mind if it is changed